server/helper.ts

```diff
@@ -304,37 +328,109 @@ function getSearchkitRouter() {
     const fullUrl = `${host}/${req.body.index || "cmmstudy_en"}${req.url}`;
     logger.debug("Start Elasticsearch Request: %s", fullUrl);
 
+    const userQuery = req.body?.[0]?.params?.query || "";
+
     try {
       const response = await apiClient.handleRequest(req.body, {
         hooks: {
           // Hook to edit search request before it is executed
           beforeSearch: async (searchRequests) => {
             return searchRequests.map((sr) => {
+              const customQuery = userQuery
+                ? {
+                  query_string: {
```

Comment:
```suggestion
                  simple_query_string: {
```

Consider using `simple_query_string` rather than `query_string`. `simple_query_string` ignores malformed queries whilst still running the search request.